publish.sh

```diff
@@ -56,4 +57,4 @@ function_name="${CUMULUS_PREFIX}-${LAMBDA_NAME}"
          --function-name "${function_name}" \
          --s3-bucket "${BUCKET}" \
          --s3-key "${KEY}") ||
-    echo "lambda function '${function_name}' not found; lambda function must be created via terraform deployment"
+    (echo "lambda function '${function_name}' not found; lambda function must be created via terraform deployment" && exit 1)
```

Comment:
Was the `&& exit 1` suggested by shellcheck, or did you add it to change how this was working?

My quick browser searches at https://gist.github.com/nicerobot/53cee11ee0abbdc997661e65b348f375 didn't lead me to a related issue page

Reply:
I'll go back and look, I'm guessing shellcheck suggested it though

Reply:
I just realized this change is from a commit I made, in October...hmm....

Reply:
OBE by rebase